### ISSUE TITLE
【UVCCamera 】Does this project also support audio output?

### ISSUE THREAD

**Issue by zondaOf2012** [saki4510t/UVCCamera#132]

**Comment by tairtec** [comment]
Do you mean UAC ?

**Comment by zondaOf2012** [comment]
@tairtec Yes. I noticed that the project use internal mic as uncompressed 16bit PCM data on the MediaAudioEncoder.

**Comment by zondaOf2012** [comment]
@tairtec Lately I am learning about read data from HDMI to USB3.0 capture to android phone , but this project seems the only surface data.
